src/FrameBuffer.cpp

```diff
@@ -1369,15 +1380,25 @@ void FrameBufferToRDRAM::_copy(u32 _startAddress, u32 _endAddress, bool _sync)
 	if (m_pCurFrameBuffer->m_size == G_IM_SIZ_32b) {
 		u32 *ptr_src = (u32*)pixelData;
 		u32 *ptr_dst = (u32*)(RDRAM + _startAddress);
-		_writeToRdram<u32, u32>(ptr_src, ptr_dst, &FrameBufferToRDRAM::_RGBAtoRGBA32, 0, 0, width, height, numPixels, _startAddress, m_pCurFrameBuffer->m_startAddress, m_pCurFrameBuffer->m_size);
+		u32 srcBuf[width * height];
```

Comment:
I'm surprised that doesn't work with MSVS. It's allowed by the language. I can change it to that.